int16 map_width
int16 map_height
float32 map_resolution
float32[] log_odds
int8[] scan_ranges
float32[] scan_angles